modules/platform/nuxeo-retention-core/src/main/java/org/nuxeo/retention/RetentionConstants.java

```diff
@@ -93,6 +93,8 @@
 
     public static final String RULE_ATTACHED_EVENT = "retentionRuleAttached";
 
+    public static final String RETENTION_CATEGORY = "Retention";
+
```

Comment:
if the add it first in master please (you did a backport) ignore my comment https://github.com/nuxeo/nuxeo/pull/3931/files#r412219197